src/Api/FacetApi.php

```diff
@@ -63,6 +94,20 @@ protected function getRequestMultipartStreamElements(): array
         return $parts;
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function setSort($sort): SearchApiBaseInterface
+    {
+        if (!in_array($sort, self::ALLOWED_SORT_VALUES)) {
+            $allowed_values = implode(', ', self::ALLOWED_SORT_VALUES);
+            throw new EuropaSearchApiInvalidParameterValueException("`setSort()` received invalid argument `$sort`, must be one of $allowed_values.");
```

Comment:
Let's pad them w/ single quotes. Also, it's a good practice to use the curly brackets when embedding vars in double quoted strings: `{$var}`.